File: cpp/pangram/pangram.cpp
#include "pangram.h"

namespace pangram {
    bool is_pangram(string input) {
        transform(input.begin(), input.end(), input.begin(), ::tolower);
        for (int i=int('a'); i<=int('z'); i++) {
            if (input.find(i) == string::npos) return false;
        }
        return true;
    }
}  // namespace pangram

File: csharp/pangram/Pangram.cs
using System;
using System.Linq;

public static class Pangram
{
    public static bool IsPangram(string input)
    {
        var chars = input.ToUpper().ToCharArray();
        foreach (var letter in "ABCDEFGHIJKLMNOPQRSTUVWXYZ".ToCharArray())
        {
            if (!chars.Contains(letter)) return false;
        }

        return true;
    }
}
